packages/supabase/scripts/db-advisor.sh

```
@@ -0,0 +1,70 @@
+# Supabase's dashboard "Security Advisor" / "Performance Advisor" is powered by
+# the open-source `splinter` lint ruleset — a single SQL query that UNION ALLs
+# ~27 checks (RLS disabled, SECURITY DEFINER views, mutable function
+# search_path, exposed auth.users, sensitive columns, etc.). The Supabase CLI
+# has NO command for this: `supabase db lint` (pnpm db:lint) is a *schema
+# typing* linter, not the advisor. So we vendor splinter.sql and run it here.
+#
+# Vendored pristine from https://github.com/supabase/splinter (splinter.sql).
+# To update: re-fetch scripts/splinter.sql from that source.
+# =============================================================================
+
+GREEN='\033[0;32m'
```

Shellcheck (0.11.0)

[warning] GREEN appears unused. Verify use (or export if used externally).

(SC2034)